src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -194,19 +216,25 @@ void LiveFunctionsDataView::DoSort() {
   std::function<bool(uint64_t a, uint64_t b)> sorter = nullptr;
 
   switch (sorting_column_) {
-    case kColumnSelected:
-      sorter = MakeFunctionSorter(
-          [this](const FunctionInfo& function_info) {
-            return app_->IsFunctionSelected(function_info);
+    case kColumnType: {
+      sorter = MakeSorter(
```

Comment:
I would add short comment describing that we order by type first, then by hooked state, then by whether a frame track is enabled. Makes it much easier to see what this block is doing without looking at the details.